WebAPI/Services/User/Authentication/UserAuthenticationService.cs

```diff
@@ -52,17 +52,17 @@ public async Task<Response> Login(LoginModel model)
                     );
 
                 var tokenResponse = new JwtSecurityTokenHandler().WriteToken(token);
+                bool _isAdmin = await _userManager.IsInRoleAsync(user, "Admin");
```

Comment:
Shouldn't you check for UserRoles.Admin instead of "Admin", as you already have that implemented elsewhere?